### ISSUE TITLE
typing.Dict, typing.List, typing.Type, ... deprecated

### ISSUE THREAD

**Issue by igorbenav** [igorbenav/FastAPI-boilerplate#89]
With `ruff check`:

```sh
src/app/api/paginated.py:1:1: UP035 `typing.Dict` is deprecated, use `dict` instead
src/app/api/paginated.py:1:1: UP035 `typing.List` is deprecated, use `list` instead
src/app/api/v1/login.py:2:1: UP035 `typing.Dict` is deprecated, use `dict` instead
src/app/api/v1/logout.py:1:1: UP035 `typing.Dict` is deprecated, use `dict` instead
src/app/api/v1/posts.py:1:1: UP035 `typing.Dict` is deprecated, use `dict` instead
src/app/api/v1/rate_limits.py:1:1: UP035 `typing.Dict` is deprecated, use `dict` instead
src/app/api/v1/tasks.py:1:1: UP035 `typing.Dict` is deprecated, use `dict` instead
src/app/api/v1/tiers.py:1:1: UP035 `typing.Dict` is deprecated, use `dict` instead
src/app/api/v1/users.py:1:1: UP035 `typing.Dict` is deprecated, use `dict` instead
src/app/core/security.py:2:1: UP035 `typing.Dict` is deprecated, use `dict` instead
src/app/core/setup.py:1:1: UP035 `typing.Dict` is deprecated, use `dict` instead
src/app/core/utils/cache.py:5:1: UP035 `typing.Dict` is deprecated, use `dict` instead
src/app/core/utils/cache.py:5:1: UP035 `typing.List` is deprecated, use `list` instead
src/app/core/utils/cache.py:5:1: UP035 `typing.Tuple` is deprecated, use `tuple` instead
src/app/crud/crud_base.py:2:1: UP035 `typing.Dict` is deprecated, use `dict` instead
src/app/crud/crud_base.py:2:1: UP035 `typing.List` is deprecated, use `list` instead
src/app/crud/crud_base.py:2:1: UP035 `typing.Type` is deprecated, use `type` instead
src/app/crud/helper.py:1:1: UP035 `typing.List` is deprecated, use `list` instead
src/app/crud/helper.py:1:1: UP035 `typing.Type` is deprecated, use `type` instead
Found 19 errors.
```

**Comment by igorbenav** [comment]
Hey, @mithun2003
Here's a low hanging fruit if you're interested.

**Comment by eredden** [comment]
Created pull request https://github.com/igorbenav/FastAPI-boilerplate/pull/90 as a fix for this issue. Re-scanning the repository with `ruff check` no longer results in any deprecation errors coming up.

**Comment by igorbenav** [comment]
closed by #90
